zp-relayer/endpoints.ts

```diff
@@ -278,6 +278,12 @@ function getParamsHash(type: 'tree' | 'transfer') {
   }
 }
 
+function relayerVersion(req: Request, res: Response) {
+  res.json({
+    version: config.version,
```

Comment:
I think it's better to put tag/branch and commit hash as separate values and let the UI decide how to render them